maps/vgstation.dmm

```diff
@@ -0,0 +1,14 @@
+"a" = (/turf/space,/area)
```

Comment:
Should I just delete that file? It serves no purpose from what it looks like. Its literally an empty map file.